### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by MEIZU16** [IDEA-Research/Grounded-Segment-Anything#111]
envrionment: 
python3.9
torch               1.13.1+cu117
torchaudio          0.13.1+cu117
torchvision         0.14.1+cu117

(gsa1.13) dspwasc@dspwasc:~/Public/Grounded-Segment-Anything$ export CUDA_VISIBLE_DEVICES=1
python grounding_dino_demo.py \
  --config GroundingDINO/groundingdino/config/GroundingDINO_SwinT_OGC.py \
  --grounded_checkpoint groundingdino_swint_ogc.pth \
  --input_image assets/demo1.jpg \
  --output_dir "outputs" \
  --box_threshold 0.3 \
  --text_threshold 0.25 \
  --text_prompt "bear" \
  --device "cuda"
/root/miniconda3/envs/gsa1.13/lib/python3.9/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3190.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
final text_encoder_type: bert-base-uncased
Some weights of the model checkpoint at bert-base-uncased were not used when initializing BertModel: ['cls.predictions.transform.dense.bias', 'cls.predictions.bias', 'cls.predictions.decoder.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.dense.weight', 'cls.seq_relationship.bias', 'cls.seq_relationship.weight']
- This IS expected if you are initializing BertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
_IncompatibleKeys(missing_keys=[], unexpected_keys=['label_enc.weight'])
/root/miniconda3/envs/gsa1.13/lib/python3.9/site-packages/transformers/modeling_utils.py:830: FutureWarning: The `device` argument is deprecated and will be removed in v5 of Transformers.
  warnings.warn(
/root/miniconda3/envs/gsa1.13/lib/python3.9/site-packages/torch/utils/checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

**Comment by MEIZU16** [comment]
好吧，原来在output里

**Comment by MEIZU16** [comment]
> 

This is actually not an error, the program has already processed the image and saved it to the outputs folder

**Comment by MEIZU16** [comment]
> Yes, I am aware, as I inspected it. But no bounding boxes have been created. Was this your case too?

I didn't have this situation, the image was tagged correctly

**Comment by zhw123456789** [comment]
> Yes, I am aware, as I inspected it. But no bounding boxes have been created. Was this your case too?

I think u need to delete the original image and run again.that's how i solve it
